### PR TITLE
Updates for modular RC.

### PR DESCRIPTION
**Pull request by davidraker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,157 @@ The Platform Driver agent is a special purpose agent a user can install on the p
 # Requires
 
 * python >= 3.10
-* volttron >= 10.0
-* volttron-lib-base-driver
+* volttron.core >= 2.0.0rc0
```
Comment (craigpnnl) [inline README.md]: volttron-core not .core

File: README.md [hunk 2]
```diff
@@ -26,7 +168,7 @@ Information on how to install of the VOLTTRON platform can be found
 Install and start the volttron-platform-driver.
 
 ```shell
-vctl install volttron-platform-driver --vip-identity platform.driver --start
+vctl install volttron-platform-driver --vip-identity platform.driver
```
Comment (craigpnnl) [inline README.md:171]: --start?

File: README.md [hunk 3]
```diff
@@ -35,6 +177,75 @@ View the status of the installed agent
 vctl status
 ```
 
+To communicate with devices, one or more driver interfaces will also need to be installed.
+Each interface is distributed as a library and may be installed separately using pip.
+In the current RC version of the driver, only two interfaces are fully supported:
+
+* A Fake Driver (which returns data from a csv file):
+    ```shell
+    pip install volttron-lib-fake-driver
```
Comment (craigpnnl) [inline README.md]: We don't do this any longer...should be poetry add --directory $VOLTTRON_HOME volttron-lib-fake-driver until we get a wrapper command.

File: README.md [hunk 4]
```diff
@@ -35,6 +177,75 @@ View the status of the installed agent
...
+    ```
+* BACnet:
+    ```shell
+    pip install volttron-lib-fake-driver
```
Comment (craigpnnl) [inline README.md]: I think this is the wrong pypi project.

**Review by craigpnnl — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (davidraker) [reply README.md:171]: If you start it first, and then configure, your configs won't take until you restart. Even once that is corrected, you would be rescheduling every time you add add a device to the configs. This should not have --start.

**Review by craigpnnl — APPROVED** [review]

**Pull request merged**